src/pages/documentation/adobe-campaign-standard/release-notes.md

```diff
@@ -17,4 +23,4 @@
 
 ### iOS Campaign Standard 3.0.0
 
```

Comment:
`Android Campaign 1.0.8` section should be removed from this page and moved to previous versions